token/services/interop/htlc/transaction.go

```diff
@@ -251,12 +253,6 @@ func (t *Transaction) Claim(wallet *token.OwnerWallet, tok *token2.UnspentToken,
 		return errors.New("failed to unmarshal RawOwner as an htlc script")
 	}
 
```

Comment:
let's check if the preImage matches the image in the script. We avoid possible usage mistake :)